<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulator3CControl</class>
 <widget class="QWidget" name="Simulator3CControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="simGroupBox">
     <property name="title">
      <string>Simulator3C</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Block heigh</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QDoubleSpinBox" name="globalSubmersion">
        <property name="suffix">
         <string>[mm]</string>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Skip to end</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="heightmapSize">
        <property name="minimum">
         <number>512</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="value">
         <number>2048</number>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QPushButton" name="simPlayPause">
        <property name="text">
         <string>Play/Pause</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="blockSizeX">
        <property name="prefix">
         <string notr="true">X </string>
        </property>
        <property name="suffix">
         <string>[cm]</string>
        </property>
        <property name="minimum">
         <double>2.000000000000000</double>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Block vertices</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="blockSizeY">
        <property name="prefix">
         <string>Y </string>
        </property>
        <property name="suffix">
         <string>[cm]</string>
        </property>
        <property name="minimum">
         <double>2.000000000000000</double>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tool diameter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tool type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Heightmap size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Load paths</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="toolDiameter">
        <property name="suffix">
         <string>[mm]</string>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QRadioButton" name="toolSphere">
        <property name="text">
         <string>Spherical</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="blockVerticesX">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>X </string>
        </property>
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>512</number>
        </property>
        <property name="value">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QRadioButton" name="toolCylinder">
        <property name="text">
         <string>Cylindrical</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="BlockVerticesY">
        <property name="prefix">
         <string>Y </string>
        </property>
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>512</number>
        </property>
        <property name="value">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Block size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Maximum global submersion</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="blockHeight">
        <property name="prefix">
         <string>Z </string>
        </property>
        <property name="suffix">
         <string>[cm]</string>
        </property>
        <property name="minimum">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QCheckBox" name="pathsShow">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Show paths</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Maximum tool submersion</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="toolSubmersion">
        <property name="suffix">
         <string>[mm]</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QProgressBar" name="simProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Clear paths</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Reset simulator</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
